charts/brigade-metrics/templates/exporter/deployment.yaml

```diff
@@ -32,7 +32,7 @@ spec:
               name: {{ include "brigade-metrics.exporter.fullname" . }}
               key: api-token
         - name: API_IGNORE_CERT_WARNINGS
-          value: {{ quote .Values.exporter.brigade.ignoreCertWarnings }}
+          value: {{ quote .Values.exporter.brigade.apiIgnoreCertWarnings }}
```

Comment:
`ignoreCertWarnings` doesn't match what was in the `values.yaml`. It needs to be `ignoreCertWarnings`.